imap_processing/cdf/config/imap_spacecraft_global_cdf_attrs.yaml

```
@@ -0,0 +1,20 @@
+Descriptor: &desc
+  Spacecraft>IMAP Spacecraft Data
+TEXT: &txt >
+  Spacecraft data includes all data collected by the IMAP spacecraft and
+  not specific to any one instrument. This includes quaternions from the
+  attitude control system.
+Instrument_type: Particles (space)
+
+# -------------- Product specific global attributes defined below --------------
+imap_spacecraft_l1a_quaternions_attrs:
+    Data_level: 1A
+    Data_type: L1A_QUAT>Level-1A Quaternions
+    Logical_source: imap_spacecraft_l1a_quaternions
+    Logical_source_description: IMAP Quaternion Level-1A Data From the Spacecraft.
+
+imap_spacecraft_l1b_quaternions_attrs:
```

We usually don't add `attrs` to global attrs.
```suggestion
imap_spacecraft_l1a_quaternions:
    Data_level: 1A
    Data_type: L1A_QUAT>Level-1A Quaternions
    Logical_source: imap_spacecraft_l1a_quaternions
    Logical_source_description: IMAP Quaternion Level-1A Data From the Spacecraft.

imap_spacecraft_l1b_quaternions:
```

---

Thanks, updated!